velox/vector/BaseVector.cpp

```diff
@@ -673,6 +673,13 @@ bool isLazyNotLoaded(const BaseVector& vector) {
     case VectorEncoding::Simple::CONSTANT:
       return vector.valueVector() ? isLazyNotLoaded(*vector.valueVector())
                                   : false;
+    case VectorEncoding::Simple::ROW: {
+      auto childrenVector = vector.as<RowVector>()->children();
```

Comment:
nit: "vector" is a pretty overloaded term throughout the codebase, maybe just call it "children"?